### PR TITLE
Captain Nate says: Ram them!

### PR DESCRIPTION
**Pull request by Pointedstick**
At the community's request, created an in-game Person for myself (Nate Graham/Pointedstick). Thanks for the good vibes, everyone!

Also fixed an issue with Marauding Max K's person (`government` was specified twice).

### CODE REVIEW

**Comment by Amazinite** [comment]
Give it [21 guns](https://youtu.be/r00ikilDxW4). ( ͡° ͜ʖ ͡°)	 


(Then make it say Green Day lyrics.)

**Comment by Pointedstick** [comment]
@Amazinite After all the work you've done, you'll need to make your own Person ship... and arm it accordingly!

File: data/persons.txt [hunk 1]
```diff
@@ -214,3 +213,122 @@ person "Marauding Max K"
...
+			"Stuff your ship full of devastating short-ranged weapons; it's fun!"
+			"Ram them!"
+			"Ram them again!"
+			"Accelerate to Waaagh velocity!"
+
+	phrase
+		word
+			"When you need to make your point, stick to Pointedstick Communications Inc.!"
+			"Pointedstick Communications: making the galaxy a more verbose place, one commit at a time!"
+			`Pointedstick inc.: "Sharpen your communication skills!"`
```
Comment (DingusShingleton) [inline data/persons.txt]: Capitalize Inc. Looks more legit ;)

**Comment by DingusShingleton** [comment]
I thought I liked you pointedstick. Then you packed your person full of Arnie quotes. Now I know.

**Comment by Amazinite** [comment]
@PointedStick lol, not yet. :)

**Pull request merged**